services/chaindb/postgresql/upgrader.go

```diff
@@ -1527,3 +1536,24 @@ CREATE TABLE t_fork_schedule (
 
 	return nil
 }
+
+func recreateValidators(ctx context.Context, s *Service) error {
```

Comment:
This doesn't seem like a suitable name for the function, something like `addValidatorWithdrawalCredentials` would be clearer.

Reply:
Changed